cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -49,6 +48,15 @@ THE SOFTWARE.
                     </l:ajax>
                 </f:dropdownListBlock>
             </j:forEach>
+        </d:tag>
+    </d:taglib>
+    <f:optionalBlock name="snippetizer" title="Snippet Generator" help="/${it.HELP_URL}">
+        <f:section title="Steps"/>
+        <!-- Similar to f:dropdownDescriptorSelector, but adds fallback content to block, and JENKINS-25130 adds per-selection help: -->
+        <f:dropdownList name="prototype" title="Sample Step">
+            <local:listSteps stepDescriptors="${it.getStepDescriptors(false)}"/>
+            <f:dropdownListBlock title="${%— Advanced/Deprecated —}"/>
```

Comment:
@jglick yeah if we can keep it private then we have options later.

I guess it wasn't obvious what it was about. The `Descriptor` interface method was called `isAdvanced`, yet the javadoc was also saying "or deprecated". "Deprecated or advanced" seem to me like a strange pairing when you don't have the full context. Maybe a bit more javadoc (with an example) would help that. Even with the context, "advanced" seems a bit odd to me because the step you want to hide might not be all that advanced as I'm sure there will be other reasons for hiding steps.

When you mentioned earlier that these steps should be hidden somehow, I was thinking of something like the `@InternalUseOnly` (forget the exact name but I know you know what I'm talking about) annotation. Maybe it's not appropriate to use the annotation, but the use of the term "InternamUseOnly" seems more accurate in itself.